.github/workflows/reusable_push_pr.yaml

```diff
@@ -0,0 +1,116 @@
+name: Push/PR no fake prereleases
+
+on:
+  workflow_call:
```

Comment:
I think the security workflow is missing

Reply:
oh i see that there is already the reusable used in other workflow which is triggered on cron schedule too. So it's not need to have it here